packages/headless/src/features/search/search-mappings.ts

```diff
@@ -67,6 +68,10 @@ function mapFacetRequest(
       ),
     };
   }
+  if (isFacetRequest(facetRequest)) {
```

Comment:
For posterity:
```suggestion
  // TODO: remove in v2
  if (isFacetRequest(facetRequest)) {
```

There should also be a test for that in `search-mappings.test.ts`